src/main/resources/application-rc.properties

```diff
@@ -0,0 +1,4 @@
+datarepo.gcs.bucket=jadedatarepodd_data
```

Comment:
i use the default for this parameter (broad-jade-dev-data) which is what we should be using for common tests